package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-in-viewport",
-  "version": "1.2.5",
+  "version": "1.2.6",
```

Comment:
decided to bump the version so a new "Ember 2.0" compatible version can be released. happy to undo if needed.